app/models/app_version.rb

```diff
@@ -0,0 +1,48 @@
+class AppVersion
+  include Comparable
+
+  DECIMAL_INTEGER = /(?:0|[1-9]\d*)/
```

Comment:
This regexp (used within the `VERSION_REGEX`) disallows ruby octal literal syntax. Trying to make sure this thing doesn't have surprising behavior.